web/admin/src/App.js

```diff
@@ -97,24 +97,6 @@ export default class App extends PureComponent {
       </div>
     )
   }
-    else 
```

Comment:
Not sure why this shows up as deleted in this PR, since it is already gone in `master` 🤔